admin/base/views.py

```diff
@@ -28,3 +34,43 @@ def password_reset_confirm_custom(request, **kwargs):
             user.confirmed = True
             user.save()
     return response
+
+
+class GuidFormView(FormView):
+    form_class = GuidForm
+    template_name = None
+    object_type = None
+
+    def get_guid_object(self):
+        raise NotImplementedError
+
+    def __init__(self):
+        self.guid = None
+        super(GuidFormView, self).__init__()
+
+    def get(self, request, *args, **kwargs):
+        try:
+            return super(GuidFormView, self).get(request, args, kwargs)
+        except AttributeError:
```

Comment:
This is where the view is returned, so it either has to return a filled in template or a 404 if the serializer comes back with an error. The serializer already raises the exception and returns to the `get_guid_object` early which I guess I could catch and add a new piece of string to, but ultimately it is going to the view.